indra/newview/llagent.cpp

```diff
@@ -5028,7 +5028,7 @@ void LLAgent::applyExternalActionFlags(U32 outer_flags)
                 if( delta_time > FLY_TIME
                     && delta_frames > FLY_FRAMES)
                 {
-                    setFlying(TRUE);
```

Comment:
But how could this pass the build check with the defines removed? 🙀

Reply:
They are not removed yet since not all of theHavok libs have been updated yet.

Reply:
Right, but I do that manually for every local build after the merge, just to ensure those fake bools don't sneak in.

Reply:
But since the Havok stub lib isn't updated yet, you cannot make an OS build with the typedefs removed.

Reply:
And that's why it's just a test on my non-OS local builds for now. It helped me catch a couple of leftovers before, but now it doesn't seem as reliable. There are probably more defines somewhere else.

Reply:
Depending on the platform they are defined in various external libraries, so they might be defined on various occasions in the viewer code.